src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -335,25 +335,23 @@ protected function describeEventDispatcherListeners(EventDispatcherInterface $ev
     {
         $event = array_key_exists('event', $options) ? $options['event'] : null;
 
-        $label = 'Registered listeners';
         if (null !== $event) {
-            $label .= sprintf(' for event <info>%s</info>', $event);
+            $title = sprintf('Registered Listeners for "%s" Event', $event);
```

Comment:
because the style guide says that titles should use the "title-case", which is what we use in the Symfony docs, symfony.com, etc.